specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1984,6 +2272,88 @@
         }
       }
     },
+    "SqlServerInstanceProperties": {
+      "description": "Properties of SqlServerInstance.",
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "null"
```

Comment:
>"null" [](start = 25, length = 6)

All properties are missing descriptions

Reply:
I saw existing properties in this file has description set as null. It is ok to leave description as null for the new resource type too?

Reply:
The descriptions are used directly in the generated REST documentation and SDKs.  They should be human readable and descriptive.  They also make these reviews a lot easier

Reply:
Thanks for the explanation. I will add descriptions.